vue.config.js

```
@@ -47,54 +53,293 @@
+        newArgs[0].chunksSortMode = 'auto';
+      }
+      return newArgs;
+    });
+    // Ensure modern mode is disabled to prevent legacy assets issues
+    // Get all plugin names
+    const pluginNames = Object.keys(config.plugins.entries() || {});
+
+    // Find and remove any legacy/modern plugins by name
+    pluginNames.forEach(name => {
+      const plugin = config.plugins.get(name);
+      if (plugin && plugin.constructor && plugin.constructor.name
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)